src/Context/BasicContext.php

```diff
@@ -510,6 +510,9 @@ public function iClickInTheElement($clickType, $text, $selector)
         $parentElement = $page->find('css', $selector);
         Assert::assertNotNull($parentElement, sprintf('"%s" element not found', $selector));
         $element = $parentElement->find('xpath', sprintf('//*[count(*)=0 and contains(.,"%s")]', $text));
+        if (is_null($element)) {
+            $element = $parentElement->find('xpath', sprintf('//*[text()[contains(.,"%s")]]', $text));
+        }
```

Comment:
I added this as an additional check in case these changes might break existing tests. This expression also works with other elements in another behat tests (e.g. silverstripe-admin), but I'm afraid that in some tests the selected elements will not be clickable.
See: https://stackoverflow.com/questions/3655549/xpath-containstext-some-string-doesnt-work-when-used-with-node-with-more/71255563#71255563